react-app/src/components/OverviewScreen/OverviewScreen.graphql

```diff
@@ -0,0 +1,10 @@
+query LatestCommunityStatus {
```

Comment:
This should be OverviewScreenQuery?

And we should have a OverviewScreenAPI.ts?

Can we also introduce RequestState?